sentry.client.config.ts

```diff
@@ -1,5 +1,10 @@
 import * as Sentry from '@sentry/nextjs';
 import { isNextJSSentryActivated } from '#utils/sentry';
+declare global {
+  interface Window {
+    IS_OUTDATED_BROWSER: boolean;
+  }
+}
```

Comment:
I dont see the use for this. As far as we know the whole next js bundle will not load if browser is outdated

The only scenario where sentry might log an erro is if sentry frontend (from the vite bundle)  load independantly from react

Reply:
TODO à documenter